defaults.ts

```diff
@@ -0,0 +1,85 @@
+import spec from "conventional-changelog-config-spec/versions/2.1.0/schema.json";
+
+type Defaults = Readonly<
```

Comment:
Is this a type for the defaults, or for the config generally? I wouldn't expect optional properties in a type named defaults

Reply:
You're right. I'll instead infer the type from the default object - we'll get the exact types from the defaults, down to the literal string, where possible (this is not the case for the defaults from the spec, they'll be generic `string`s etc.)